README.md

```diff
@@ -1,71 +1,83 @@
-# Deep Learning Pipelines for Apache Spark
+Deep Learning Pipelines for Apache Spark
+=============================================
+[![Build Status](https://travis-ci.org/databricks/spark-deep-learning.svg?branch=master)](https://travis-ci.org/databricks/spark-deep-learning)
 
-Deep Learning Pipelines provides high-level APIs for scalable deep learning in Python. The
```

Comment:
Keep a high-level, 1- or 2-sentence description at the top. Readers must understand immediately what this project is about.